src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
+    p Groups can be used to create team-based experiences where groups compete to achieve a goal within the experience. Using an adventure game again as an example, you could create a "Allies" group and a "Adversaries" group.
+    The ”Allies” group is provided with UI controls that gives them the ability to heal the character that the broadcaster is controlling or grant buffs to increase their chance of winning. Alternatively, the ”Adversaries” group could spawn traps or monsters to try to get in the way of winning using their controls.
 
-    p All participants start out in the default group. #[a(href='#the-gameclient') Game Clients] can re-assign participants to any group from the #[a(href='#the-gameclient') Game Client].
+    p Although all participants start out in the default group, #[a(href='#the-gameclient') Game Clients] can always re-assign participants into any group.
 
     h3#controls Controls
-    p A Control is an interactive component within a scene. #[a(href='#participants') Participants] can individually interact with the control, usually with their keyboard, mouse or controller. Interactive currently has two types of controls, but additional types may be added in the future. These two controls are Buttons and Joysticks.
+    p A Control is a clickable, interactive image within a scene. #[a(href='#participants') Participants] can interact with the control using keyboard, mouse, or controller. There are currently has two types of controls - Buttons and Joysticks. Additional control types may be added in the future.
 
     h4#buttons Buttons
-    .figure
-        img(src='/img/reference/interactive/button.png' alt='A button control as displayed to a participant. This button costs one spark to press.')
-    p.caption A button control as displayed to a participant. This button costs one spark to press.
-    p Buttons are a rectangle positioned within a scene that a participant can click with their mouse or trigger with their keyboard or controller. #[a(href='#the-gameclient') Game Clients] receive events when a button is interacted with:
-
+    
+    p A button is a type of control. They are rectangular, clickable interactive images within a scene. #[a(href='#the-gameclient') Game Clients] receive events when participants interact with a button.
+    
     ul
-        li `mousedown` - Emitted when a button is pressed by a participant.
-        li `mouseup` - Emitted when a button is released by a participant.
-
-    p Interactive developers can use buttons to allow participants to contribute to votes, cause actions to happen or even control in-game entities directly.
+        li `Down` event is sent when a button is pressed by a participant
```

Comment:
no, the events are `mousedown` and `mouseup`. Stuff in backticks(`) is usually code